splice_spark2/src/main/spark2.2/com/splicemachine/spark2/splicemachine/SpliceJDBCUtil.scala

```diff
@@ -0,0 +1,212 @@
+package com.splicemachine.spark2.splicemachine
```

Comment:
Copyright?  Not a blocker for bringing this in, but should be added at some point.